### PR TITLE
blog: Running Flotta on Raspberry Pi

### PR DESCRIPTION
**Pull request by jakub-dzon**
This blog post depends on https://github.com/project-flotta/flotta-operator/pull/193.

### CODE REVIEW

File: _posts/2022-04-15-flotta-and-raspberry-pi.markdown
Comment (jakub-dzon) [reply _posts/2022-04-15-flotta-and-raspberry-pi.markdown]: Done.

**Review by masayag — APPROVED** [review]
Very nice!

**Review by pkliczewski — APPROVED** [review]

File: _posts/2022-04-15-flotta-and-raspberry-pi.markdown
Comment (eloycoto) [reply _posts/2022-04-15-flotta-and-raspberry-pi.markdown:108]: :+1:

**Review by eloycoto — COMMENTED** [review]
Overall LGTM, but I'll wait until device approval to merge  to update to the right path.

**Comment by jakub-dzon** [comment]
@eloycoto - how will that change the contents of this post?

**Review by machacekondra — APPROVED** [review]

**Comment by eloycoto** [comment]
At the ends, nothing.. if auto-approve is enabled, so can be merged.

**Pull request merged**
